src/main/resources/locale_es_ES.properties

```
@@ -0,0 +1,196 @@
+command.repSettings.sub.reaction.get.emote=La reputación actual El emote se establece en: %EMOTE%
+command.repSettings.sub.reaction.get.error=No se pudo encontrar el emote en el gremio.
+command.repSettings.sub.reaction.set.emoji=El emoji de reputación está establecido en %EMOJI%
+command.repSettings.sub.reaction.set.emote=Emote de reacción establecido en %EMOTE%.
+command.repSettings.sub.reactions=Obtener o establecer si las reacciones de reputación deben dar reputación.
+command.repSettings.sub.reactions.false=El robot no procesará las reacciones.
+command.repSettings.sub.reactions.true=El robot procesará las reacciones.
+command.reputation.description=Información sobre tu reputación o la de otros.
+command.reputation.profile.nextLevel=Nivel siguiente
+command.reputation.profile.title=Reputación de %NAME%
+command.reputation.sub.top=Obtener la lista superior o una página de la lista
+command.reputation.sub.top.ranking=Ranking de %GUILD%
+command.roles.description=Gestionar los roles de reputación.
+command.roles.sub.add=Añadir un rol de reputación.
+command.roles.sub.add.added=Añadir rol %ROLE% con reputación %POINTS%.
+command.roles.sub.list=Lista de roles de reputación.
+command.roles.sub.managerRole=Cambiar el rol de manager. Los usuarios con este rol pueden gestionar el bot.
+command.roles.sub.managerRole.current=El rol de gestor actual es %ROLE%.
+command.roles.sub.managerRole.noRole=No se ha definido ningún rol de gestor.
+command.roles.sub.managerRole.set=El rol de gestor definido es %ROLE%.
+command.roles.sub.remove=Eliminar un rol de reputación.
+command.roles.sub.remove.notARepRole=Este rol no es un rol de reputación.
+command.roles.sub.remove.removed=Eliminar rol %ROLE%.
+command.scan.canceled=Escaneo cancelado.
+command.scan.canceling=Intentando cancelar el escaneo.
+command.scan.completed=Escaneo completado.
+command.scan.description=Escanear un canal en busca de reputaciones.
+command.scan.error.history=Falta de permiso para el historial de mensajes.
+command.scan.error.queueFull=Se están ejecutando demasiados escaneos actualmente. Por favor, espere hasta que haya un espacio libre.
+command.scan.error.running=Se está ejecutando una tarea en este gremio. Por favor, espere a que termine.
+command.scan.progress=Progreso: %PERCENT%
+command.scan.result=Mensajes escaneados %SCANNED%.\nSe ha añadido reputación %HITS%.
+command.scan.scheduling=Programando el escaneo. El tiempo estimado necesario hasta la finalización es: %DURATION%
+command.scan.sub.cancel=Cancelar una exploración en curso.
+command.scan.sub.cancel.noTask=No hay ninguna tarea en ejecución para este gremio.
+command.setup.description=Realizar una configuración básica del bot.
+command.setup.dialog.channels=Introducir todos los canales en los que los usuarios pueden recibir reputación.
+command.setup.dialog.continueToProceed=Por favor, pulsa "$palabra.continue$" para continuar.
```

A key has been translated
```suggestion
command.setup.dialog.continueToProceed=Por favor, pulsa "$word.continue$" para continuar.
```